### DISCUSSION TITLE
Error with addDoubletScores

### DISCUSSION THREAD

**Discussion by libell1** [GreenleafLab/ArchR#1714] [Questions / Documentation]
Hello,

I got an error when trying to use addDoubleScores(). It says I'm "computing too large a percentage of total singular values, use a standard svd instead." It also did not compute the doublet score for one of my samples. How can I tell ArchR to not use one of the Arrow Files? I attached the log file below, but please let me know if it didn't work...

Thank you!

[ArchR-addDoubletScores-c1934c39a6fa-Date-2022-11-02_Time-15-19-48.log](https://github.com/GreenleafLab/ArchR/files/9924731/ArchR-addDoubletScores-c1934c39a6fa-Date-2022-11-02_Time-15-19-48.log)

**Comment by rcorces** [comment]
Error reports should be made to the "Issues" section.
Please confirm whether you are able to run the code from the manual on the tutorial data.

> How can I tell ArchR to not use one of the Arrow Files?

The `input` parameter accepts a character vector of Arrow Files so you can specify which to run but its unclear from what you've provided why that one sample is causing a problem for you.

https://www.archrproject.com/reference/addDoubletScores.html

**Comment by libell1** [comment]
Sorry, new to GitHub and did not see this question asked in the "Closed" section of "Issues." Will move my question to that thread!

**Comment by sylestiel** [comment]
Hi @rcorces ,
   Tried it on different computers. Same problem encountered. Tried re-installing irlba as well as using threads=1.  Can you help resolve this issue.


[ArchR-addDoubletScores-8f73bb4aad4-Date-2023-12-13_Time-19-15-21.144885.log](https://github.com/GreenleafLab/ArchR/files/13667521/ArchR-addDoubletScores-8f73bb4aad4-Date-2023-12-13_Time-19-15-21.144885.log)

**Comment by sylestiel** [comment]
@rcorces 
finally worked. Please close.

**Reply by hira-anis** [reply]
Hey, how were you able to solve the issue with Matrix/irlba?
